source/javascripts/components/GuidedOnboarding/GuidedOnboarding.tsx

```diff
@@ -85,12 +87,16 @@ export const GuidedOnboarding = ({
     isEnabled = false,
     isOpen = false,
     onTurnOff,
-    buildStatus
+    buildStatus,
+    lastWorkflowEditedDate,
+    lastRunningBuildFinishDate,
 }: GuidedOnboardingProps): JSX.Element | null => {
   const { isMobile } = useResponsive();
 
   const stepSuccessful = {
-    run_a_build: buildStatus !== BuildStatus.Running
+    run_a_build: buildStatus !== BuildStatus.Running,
+    config_workflows: (!!lastRunningBuildFinishDate && !!lastWorkflowEditedDate) 
+                        && (new Date(lastRunningBuildFinishDate) > new Date(lastWorkflowEditedDate)),
```

Comment:
As with the website PR, I don't see where you're checking that the build was successful.

Reply:
This has been updated.